pytest.ini

```diff
@@ -9,7 +9,7 @@ addopts =
     # show values of the local vars in errors:
     --showlocals
     # coverage reports
-    --cov=aiohttp_admin/ --cov=tests/ --cov-report term
```

Comment:
Together with this, add `source_pkgs = aiohttp_admin` and `source = tests` to `.coveragerc`.